devops/resources/saml-ssl-expiration-check.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+#we don't use u in the pipefail because Jenkins doesn't set variables and we want to check for that
+set -exo pipefail
+
+HOME=/edx/var/jenkins
+
+env
+set -x
+
+cd $WORKSPACE/configuration
+pip install -r requirements.txt
+. util/jenkins/assume-role.sh
+
+assume-role ${ROLE_ARN}
+
+SAML_SSL_CERT_FILE=$WORKSPACE/configuration_secure/ansible/vars/${SAML_CERT_FILE}
```

Comment:
This should be `$WORKSPACE/configuration_secure/${SAML_CERT_FILE}` right?  Since the whole path is passed in?